pt-BR/mail/chrome/messenger/credits.dtd

```diff
@@ -11,19 +11,19 @@
 -->
 <!ENTITY credit.translation "<h3>Versão brasileira</h3><ul><li>http://br.mozdev.org</li></ul>">
 
-<!ENTITY credit.visuals     "Visual Design Coordinator">
+<!ENTITY credit.visuals     "Coordenador de Design Visual">
 <!ENTITY credit.theme       "Design do tema">
-<!ENTITY credit.brand       "Brand Identity">
+<!ENTITY credit.brand       "Identidade da Marca">
 <!ENTITY credit.web         "Web Design">
-<!ENTITY credit.update      "Mozilla Update">
-<!ENTITY credit.qalead      "Quality Assurance Lead">
-<!ENTITY credit.qa          "Qualidade assegurada">
-<!ENTITY credit.build       "Build e release">
-<!ENTITY credit.infra       "Infrastructure Support">
-<!ENTITY credit.support     "Support Resources">
-<!ENTITY credit.manage      "Project Management">
-<!ENTITY credit.marketleads "Marketing Leads">
+<!ENTITY credit.update      "Atualização do Mozilla">
+<!ENTITY credit.qalead      "Líderes de Garantia de Qualidade">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY credit.qalead      "Responsável pela Qualidade">